packages/engine/src/simulation/package/init/packages/js_py/package.js

```diff
@@ -88,14 +90,18 @@ const run_task = (experiment, _sim, task_message, _group_state, context) => {
 
     let data;
     try {
-        data = JSON.stringify(agents);
+        data = {
+            "SuccessMessage": {
+                "agent_json": JSON.stringify(agents)
+            }
+        };
```

Comment:
We should revisit this at some point to remove the need for `JSON.stringify` here.

Reply:
When we were working through it I thought the same, just realised an easy solution, have updated the PR